docs/synapse/userguides/storm_ref_cmd.rstorm

```diff
@@ -68,48 +85,58 @@ This section details the usage and syntax for built-in Storm commands. Many of t
 - `tree`_
 - `trigger`_
 - `uniq`_
+- `uptime`_
 - `version`_
 - `view`_
 - `wget`_
 
 See :ref:`storm-ref-syntax` for an explanation of the syntax format used below.
 
-The Storm query language is covered in detail starting with the :ref:`storm-ref-intro` section of the Synapse User Guide.
+The Storm query language is covered in detail starting with the :ref:`storm-ref-intro` section of the
+Synapse User Guide.
 
-.. NOTE::
+.. TIP::
 
-  Storm commands, including custom commands, are added to the Cortex as **runtime nodes** ("runt nodes" - see :ref:`gloss-node-runt`) of the form ``syn:cmd``. These runt nodes can be lifted and filtered just like standard nodes in a Cortex.
+  Storm commands, including custom commands, are added to Synapse as **runtime nodes** ("runt nodes"
+  - see :ref:`gloss-node-runt`) of the form ``syn:cmd``. With a few restrictions, these runt nodes
+  can be lifted, filtered, and operated on similar to the way you work with other nodes.
 
 **Example**
 
 Lift the ``syn:cmd`` node for the Storm ``movetag`` command:
 
-
 .. storm-cli:: syn:cmd=movetag
 
+
 .. NOTE::
 
-  While the documentation for built-in Storm commands is relatively basic, the ``syn:cmd`` form within the data model includes additional secondary properties that can be used to make commands (particularly custom commands) more "self-documenting" to users, because detail about the command can be introspected directly within the data model from within Storm. For example, the ``:input`` and ``:output`` properties can be used to specify a list (:ref:`type-array`) of forms that can be input (submitted) to the command and a list of forms that may be created by the command, respectively.
+  The ``syn:cmd`` form within the Synapse data model includes secondary properties that can optionally be
+  used to make commands (particularly custom commands) more "self-documenting" to users, because detail
+  about the command can be introspected directly within Synapse. For example, the ``:input`` and ``:output``
```

Comment:
We generally don't use the `:input` and `:output` values anymore (maybe should deprecate them to reflect that?)"

Reply:
Created VTX-1745 to discuss.

Reply:
Since the props are read only on the runt nodes (we're not setting them, and user's can't set them), removed the paragraph.